site/cds_rdm/legacy/resolver.py

```diff
@@ -49,3 +53,12 @@ def get_record_by_version(parent_pid_value, version):
         # If record is not found, that means the version doesn't exist
         raise VersionNotFound(version=version, latest_record=latest_record)
     return hits[0]
+
+
+def get_community_by_uuid(community_uuid):
```

Comment:
This is not needed and we should rely on the db instead.
You need to use [the read service method](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/services/service.py#L664), see an [example here](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/resources/resource.py#L119).

The identity should be system process but... pay attention that the community might be restricted. So we cannot leak any info. I would add a comment.
A redirection works because the permissions will be checked.

Reply:
Okay! Why do we want to keep it `system_identity`?

Reply:
because we won't be able to read from the service to obtain the slug. By the way, why not returning the slug only? it would be safer than returning the whole community item